### PR TITLE
Creating accessible form exercise

### PR DESCRIPTION
**Pull request by augusticor**
Accessible form with validations

### CODE REVIEW

File: 5-Accessibility/accessible-form.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta name="author" content="augusticor">
+        <link rel="stylesheet" href="styles.css">
+        <title>Document</title>
+    </head>
+
+    <body>
+        <form action="">
```
Comment (juancho11gm) [inline 5-Accessibility/accessible-form.html]:
```suggestion
        <form>
```

File: 5-Accessibility/accessible-form.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            <fieldset>
+                <legend>Please enter your contact details</legend>
+                <label for="name">
+                    Your name: <input type="text" name="name-txt" id="name" placeholder="your name" required />
+                    <abbr title="required name input" aria-label="required">*</abbr>
```
Comment (juancho11gm) [inline 5-Accessibility/accessible-form.html:17]: Nice to see the `aria-attributes`

File: 5-Accessibility/accessible-form.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                <label for="other">
+                    <input type="checkbox" name="other" id="other" /> Other
+                </label>
+            </fieldset>
+
+            <p>Please enter a message</p>
+            <textarea name="textarea" id="texta" cols="60" rows="10" aria-label="optional message"></textarea>
+
+            <div>
+                <button type="submit">Send message</button>
```
Comment (juancho11gm) [inline 5-Accessibility/accessible-form.html]:
```suggestion
                <button>Send message</button>
```

File: 5-Accessibility/styles.css [hunk 4]
```diff
@@ -0,0 +1,7 @@
+fieldset {
+    display: flex;
```
Comment (juancho11gm) [inline 5-Accessibility/styles.css:2]: nice to see `flexbox`. We are going to see it deeper when we touch CSS.

**Review by juancho11gm — CHANGES_REQUESTED** [review]
Nice job Oscar. I see that you get the concept. Please see the comments and send the changes. You can remove the empty spaces in the file.

**Comment by augusticor** [comment]
### I've made the changes

**Comment by juancho11gm** [comment]
Recommendation: add the keyword and description meta tags in the `<head>`.

**Pull request merged**